send/generic_send

```diff
@@ -121,9 +130,10 @@ fi
 
 if [ -d "$SERVICE_FILES_FOR_DESTINATION" ]
 then
-	tar -c -C "$SERVICE_FILES_FOR_DESTINATION" . -C "$SERVICE_FILES_DIR"  --exclude="_destination" .  -C "$TMP_HOSTNAME_DIR" . | timeout -k $TIMEOUT_KILL $TIMEOUT $TRANSPORT_COMMAND
+	tar $TAR_MODE -C "$SERVICE_FILES_FOR_DESTINATION" . -C "$SERVICE_FILES_DIR"  --exclude="_destination" .  -C "$TMP_HOSTNAME_DIR" . | timeout -k $TIMEOUT_KILL $TIMEOUT $TRANSPORT_COMMAND
 else
-	tar -c -C "$SERVICE_FILES_DIR"  --exclude="_destination" .  -C "$TMP_HOSTNAME_DIR" . | timeout -k $TIMEOUT_KILL $TIMEOUT $TRANSPORT_COMMAND 
+	# Send a gziped tar archive via HTTP(s)
```

Comment:
I guess this comment should be either on both sending modes (per-destination/all destinations) or better at line 121, where you enforce ``TAR_MODE`` for URL destination type. Code seems OK.